### PR TITLE
Post index

### PR DESCRIPTION
**Pull request by LouisBrizuela**

### CODE REVIEW

File: src/__tests__/PostIndex.test.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        </BrowserRouter>
+      )
+      mockPosts.forEach((post) => {
+            const altTxt = screen.getByRole('heading', {
+                name: /title: post1/i
+            })
+            expect(altTxt).toBeInTheDocument
+        })
+    })
+})
```
Comment (SunkissedQueen) [inline src/__tests__/PostIndex.test.js]: looks like a mixture of 2 and 4 spaces

File: src/mockPosts.js [hunk 2]
```diff
@@ -1,11 +1,25 @@
...
+            user_id: 2,
+            id: 2,
+            title: "post2",
+            userName: "legoLover321",
+            timeSpent: "2 hours",
+            price: "$500",
+            review: "test review 2",
+            difficulty: 2,
+            image: "https://images.unsplash.com/photo-1585366119957-e9730b6d0f60?auto=format&fit=crop&q=60&w=800&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8Mnx8bGVnb3xlbnwwfHwwfHx8MA%3D%3D"
+        }
```
Comment (SunkissedQueen) [inline src/mockPosts.js]: looks like these are indented too much to the right

File: src/pages/PostIndex.js [hunk 3]
```diff
@@ -1,9 +1,45 @@
...
+                                See more details
+                                </NavLink>
+                            </Button>
+                        </CardBody>
+                    </Card>
+                </div>
+          )
+        })}
+      </main>
+    </>
```
Comment (SunkissedQueen) [inline src/pages/PostIndex.js]: check the indentation

**Review by SunkissedQueen — CHANGES_REQUESTED** [review]
Just some minor fixes

File: src/pages/PostIndex.js [hunk 4]
```diff
@@ -3,41 +3,42 @@ import { Card, CardBody, CardTitle, CardSubtitle, CardImg, Button } from "reacts
...
+                                    See more details
+                                    </NavLink>
+                                </Button>
+                            </CardBody>
+                        </Card>
+                    </div>
+                )
+            })}
+        </main>
     </>
```
Comment (SunkissedQueen) [inline src/pages/PostIndex.js:42]: just need to indent all of this to the right so everything is nested within the ()

**Review by SunkissedQueen — APPROVED** [review]
please see the last comment about PostIndex page

File: src/pages/PostIndex.js (hunk 4 above)
Comment (LouisBrizuela) [reply src/pages/PostIndex.js:42]: i cant believe i missed that, thank you

**Pull request merged**